src/components/structures/MatrixChat.tsx

```diff
@@ -350,10 +350,8 @@ export default class MatrixChat extends React.PureComponent<IProps, IState> {
 
         if (SettingsStore.getValue("analyticsOptIn")) {
             Analytics.enable();
-            CountlyAnalytics.instance.enable(false);
```

Comment:
Hmm, so was it backwards on purpose before...? (It used to say `false` when opted _in_...?)

Reply:
the parameter is `anonymous=true` so when opted-in it'd say false, yes. But at this part of the app lifecycle we haven't yet got a MatrixClient so can't generate the pseudonymous countly userId yet